docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md

```diff
@@ -44,6 +44,11 @@ sudo apt -y install clang-7 libssl-dev gdb libsgx-enclave-common libsgx-enclave-
 > package which is necessary for performing remote attestation in Azure. A general
 > implementation for using Intel DCAP outside the Azure environment is coming soon.
 
+If you wish to use clang-8 or clang-9, also install them
```

Comment:
I see you updated this file but not https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md

Do clang-8 and clang-9 not work for Ubuntu 16.04 or should that file be updated?

Reply:
I would actually not add this here as it complicates the flow. I think it's fair to document the set of clang versions we support via the CMakeLists.txt check. Everywhere else we should simplify by the providing the single standard path; previously we would test against the highest version supported and recommend that in all instructions.